docs/guides/getting-started-orchestrate-apis.md

```diff
@@ -10,11 +10,18 @@ keywords:
 <span class="badge badge--beginner">Beginner</span>
 <span class="badge badge--medium">Time estimate: 15 minutes</span>
 
-A Connector is a reusable building block that works out of the box. Each Connector task can be configured with domain-specific parameters without implementing custom business logic.
+This guide will walk you through working with a REST Connector task as a first time Camunda 8 user.
```

Comment:
I would expand on this - mention the REST Connector is a protocol connector, and link to it. I would also add a link to the OOTB Connectors in case folks looking for more general Connnectors. This is just one Connector, but it's the Connector for API orchestration.